docs/ref/contrib/gis/functions.txt

```diff
@@ -0,0 +1,443 @@
+``precision``          Not used on Oracle. It may be used to specify the number
+                       of significant digits for the coordinates in the GML
+                       representation -- the default value is 8.
+
+``version``            Not used on Oracle. It may be used to specify the GML
+                       version used, and may only be values of 2 or 3.  The
+                       default value is 2.
+=====================  =====================================================
+
+__ http://en.wikipedia.org/wiki/Geography_Markup_Language
+
+AsKML
+-----
+
+.. class:: AsKML(expression, precision=8, **extra)
+
+*Availability*: PostGIS, SpatiaLite
+
+Accepts a single geographic field or expression and returns a `Keyhole Markup
+Language (KML)`__ representation of the geometry.
+
+Example::
+
+    >>> qs = Zipcode.objects.annotate(kml=AsKML('poly'))
+    >>> print(qs[0].kml)
+    <Polygon><outerBoundaryIs><LinearRing><coordinates>-103.04135,36.217596,0 ...
+    -103.04135,36.217596,0</coordinates></LinearRing></outerBoundaryIs></Polygon>
+
+=====================  =====================================================
+Keyword Argument       Description
+=====================  =====================================================
+``precision``          This keyword may be used to specify the number of
+                       significant digits for the coordinates in the KML
+                       representation -- the default value is 8.
+=====================  =====================================================
+
+__ https://developers.google.com/kml/documentation/
+
+AsSVG
+-----
+
+.. class:: AsSVG(expression, relative=False, precision=8, **extra)
+
+*Availability*: PostGIS, SpatiaLite
+
+Accepts a single geographic field or expression and returns a `Scalable Vector
+Graphics (SVG)`__ representation of the geometry.
+
+=====================  =====================================================
+Keyword Argument       Description
+=====================  =====================================================
+``relative``           If set to ``True``, the path data will be implemented
+                       in terms of relative moves. Defaults to ``False``,
+                       meaning that absolute moves are used instead.
+
+``precision``          This keyword may be used to specify the number of
+                       significant digits for the coordinates in the SVG
+                       representation -- the default value is 8.
+=====================  =====================================================
+
+__ http://www.w3.org/Graphics/SVG/
+
+BoundingCircle
+--------------
+
+.. class:: BoundingCircle(expression, num_seg=48, **extra)
+
+*Availability*: PostGIS
+
+Accepts a single geographic field or expression and returns the smallest circle
+polygon that can fully contain the geometry.
+
+See also http://postgis.net/docs/ST_MinimumBoundingCircle.html
```

Comment:
maybe we could link the "PostGIS" text next to "Availability" (prefer avoiding raw URLs in the text)